### DISCUSSION TITLE
About "We are boycotting The Perl Foundation"

### DISCUSSION THREAD

**Discussion by yuki-kimoto** [mojolicious/mojo#1808] [General]
I'm worried and post this message.

Is there anything I can help?

**Comment by MartinMcGrath** [comment]
Context: https://twitter.com/kraih/status/1408525949218570240

**Comment by yuki-kimoto** [comment]
I see the topic. 

I've contributed to Mojolicious in the past and current, so can I help  to solve this problem?
